docs/overview/performance/l2ethswitch.rst

```diff
@@ -0,0 +1,18 @@
+ .. _l2ethswitch:
+
+.. toctree::
+
+L2 Ethernet Switching Performance Tests
+***************************************
+
```

Comment:
I wouldn't show "Further information..." as a note just text is good. I would also move it to the page above this.